app/modules/tables/views/EditView.js

```diff
@@ -210,8 +210,9 @@ define([
 
         if (action === 'save-form-add') {
           goToNewItem();
-        } else {
+        } else if (action !== 'save-form-stay') {
```

Comment:
Hey @danhipke curious what does this actually solve?

Reply:
@WellingGuzman To reproduce:

1. Go to the Table view and click an item to edit.
2. Make no changes to the item. Click Save and Stay.

Expected result is that you stay in the item editing view. Actual result (before this fix) is that you are taken back to the table view.